tests/tests/swfs/from_avmplus/as3/Math/e15_1_2_2/Test.as

```diff
@@ -0,0 +1,34 @@
+/* -*- Mode: js; c-basic-offset: 4; indent-tabs-mode: nil; tab-width: 4 -*- */
+/* vi: set ts=4 sw=4 expandtab: (add to ~/.vimrc: set modeline modelines=5) */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package {
+import flash.display.MovieClip;
+public class Test extends MovieClip {}
+}
+
+import com.adobe.test.Assert;
+
+// var SECTION = "15.1.2.2 parseInt()";
+// var VERSION = "";
+// var TITLE = "Tests based on code coverage";
+
+
+var testcases = getTestCases();
+
+function getTestCases() {
+    var array = new Array();
+    var item = 0;
+
+
+    // other js engines produce 1228503247364293600 for this testcase, and c produces 1228503247364293699
```

Comment:
I'm not sure what to do with this case, as Flash Player fails it (both on Linux and Windows), but Ruffle passes :thinking: